### PR TITLE
TMA-1033: report pid in case of UFB failure

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: lib/gooddata/models/user_filters/user_filter_builder.rb [hunk 1]
```diff
@@ -390,8 +390,13 @@ def self.execute_mufs(user_filters, options = {})
       project_users = project.users
       filters = normalize_filters(user_filters)
       user_filters, errors = maqlify_filters(filters, project_users, options.merge(users_must_exist: users_must_exist, type: :muf))
+      if !ignore_missing_values && !errors.empty?
+        errors = errors.map do |e|
+          e.merge(pid: project.pid)
```
Comment (yenkins) [inline lib/gooddata/models/user_filters/user_filter_builder.rb]: info: Calls 'project.pid' 5 times (DuplicateMethodCall) (reek)

File: lib/gooddata/models/user_filters/user_filter_builder.rb [hunk 2]
```diff
@@ -390,8 +390,13 @@ def self.execute_mufs(user_filters, options = {})
       project_users = project.users
       filters = normalize_filters(user_filters)
       user_filters, errors = maqlify_filters(filters, project_users, options.merge(users_must_exist: users_must_exist, type: :muf))
+      if !ignore_missing_values && !errors.empty?
+        errors = errors.map do |e|
```
Comment (yenkins) [inline lib/gooddata/models/user_filters/user_filter_builder.rb]: info: Has the variable name 'e' (UncommunicativeVariableName) (reek)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/597/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1419/ : SUCCESS in 2m 10s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1689/ : SUCCESS in 3m 38s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/638/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1467/ : SUCCESS in 2m 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1728/ : FAILURE in 1m 07s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/639/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1468/ : FAILURE in 1m 40s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1729/ : SUCCESS in 2m 37s

**Comment by Hahihula** [comment]
extended test

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/715/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1542/ : SUCCESS in 40m 42s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/103/ : FAILURE in 1m 17s

**Comment by panjan** [comment]
Why do we add the project id ad hoc? Shouldn't it be done here? https://github.com/gooddata/gooddata-ruby/blob/master/lib/gooddata/models/user_filters/user_filter_builder.rb#L231

**Pull request closed without merging**